<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagedTableBar</class>
 <widget class="QWidget" name="PagedTableBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>81</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <spacer name="horizontalLeftSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="upPageButton">
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="downPageButton">
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="currPage">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="gotoLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="gotoButton">
      <property name="text">
       <string>Goto</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalRightSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
